tests/pipeline/test_pipeline.py

```diff
@@ -240,12 +240,51 @@ def test_with_custom_config(self, tmpdir, snapshot):
 
         enriched_pipeline = load_yaml_file(Path(output_file_path))
         assert isinstance(enriched_pipeline, dict)
-        for app_details in enriched_pipeline["components"]:
-            output_topic = app_details["app"]["streams"]["outputTopic"]
-            assert output_topic == "random-topic"
-            app_type = app_details["type"]
-            if app_type == "streams-app":
-                error_topic = app_details["app"]["streams"]["errorTopic"]
-                assert error_topic == "random-error"
+
+        producer_details = enriched_pipeline["components"][0]
+        output_topic = producer_details["app"]["streams"]["outputTopic"]
+        assert output_topic == "random-topic"
+
+        streams_app_details = enriched_pipeline["components"][1]
+        output_topic = streams_app_details["app"]["streams"]["outputTopic"]
+        assert output_topic == "random-topic"
```

Comment:
So the output topic is the same for both steps?

Reply:
yes i declared it [here](https://github.com/bakdata/kpops/blob/feat/default-topics/tests/pipeline/resources/custom-config/config.yaml#L4)

But if this might cause some confusions, i could change it to `"${component_name}-test-topic"`